release-notes/1.x.md

```diff
@@ -16,6 +16,15 @@ The `usernameAwareTokenRequestOAuth2AuthorizedClientManager` needs to be defined
 at `@RequestScope` so that it can access the current authorisation each time.  The reactive version
 is `reactiveUsernameAwareTokenRequestOAuth2AuthorizedClientManager`.
 
+## Dependency version updates:
```

Comment:
I've only specified the dependencies here that other projects will inherit as a result of including the libraries